apps/designer/app/designer/features/style-panel/controls/combobox-control.tsx

```diff
@@ -0,0 +1,145 @@
+import { KeyboardEvent } from "react";
+import type { StyleValue } from "@webstudio-is/react-sdk";
+import {
+  Flex,
+  Grid,
+  Text,
+  Combobox,
+  IconButton,
+  TextField,
+  Tooltip,
+} from "@webstudio-is/design-system";
+import { PropertyName } from "./property-name";
+import { getFinalValue } from "../shared/get-final-value";
+import { useIsFromCurrentBreakpoint } from "../shared/use-is-from-current-breakpoint";
+import { ControlProps } from "../style-sections";
+import { RowGap, ColumnGap } from "@webstudio-is/icons";
```

Comment:
I just noticed all other icons have Icon in name, e.g. BrushIcon, should we stay consistent?